src/Symfony/Component/VarDumper/Exception/ThrowingCasterException.php

```diff
@@ -17,11 +17,15 @@
 class ThrowingCasterException extends \Exception
 {
     /**
-     * @param callable   $caster The failing caster
-     * @param \Exception $prev   The exception thrown from the caster
+     * @param \Exception $prev The exception thrown from the caster
      */
-    public function __construct($caster, \Exception $prev)
+    public function __construct($prev, \Exception $e = null)
     {
+        if (!$prev instanceof \Exception) {
+            @trigger_error('Providing the $prev exception as 2nd argument when instanciating a '.__CLASS__.' is deprecated since version 2.8 and will be removed in 3.0. Provide it as first argument instead.', E_USER_DEPRECATED);
```

Comment:
Isn't the real change that you should not pass the caster anymore (which implies that the exception is now the first argument)?